solvation_analysis/tests/conftest.py

```diff
@@ -89,3 +98,41 @@ def atom_groups(u_real):
     fec_atoms = u_real.atoms.select_atoms("byres type 21")
     atom_groups = {"li": li_atoms, "pf6": pf6_atoms, "bn": bn_atoms, "fec": fec_atoms}
     return atom_groups
+
+
+@pytest.fixture
+def rdf_bins_and_data_easy():
```

Comment:
The following three are very similar / seem to be using the same code. Can you either a) parametrize the fixture, or b) just create a helper function that just takes in the relevant bins & data references and returns the dictionaries?

Reply:
Seconded.

Reply:
Done